mod-orders/schemas/composite_purchase_order.json

```diff
@@ -12,6 +12,16 @@
       "description": "whether or not the purchase order has been approved",
       "type": "boolean"
     },
+    "approvedBy": {
+      "description": "UUID of the user approving the order",
+      "type": "string",
```

Comment:
Please use `"$ref": "../../common/schemas/uuid.json"` to have this pattern defined in one place (at least for new changes)

Reply:
Done, Also modified the field name to have a suffix of Id